templates/base.html.php

```diff
@@ -4,7 +4,7 @@
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <meta name="description" content="RSS-Bridge" />
-    <title><?= e($title ?? 'RSS-Bridge') ?></title>
+    <title><?= e($_title ?? 'RSS-Bridge') ?></title>
```

Comment:
Maybe `$siteTitle` or `$instanceName`?

Reply:
Good suggestions. I like to use underscore e.g. `$_foobar` for **internal variables** in templates. The goal is to avoid variable name collision e.g. `render('foo.tpl', ['title' => 'this is h1 title, dont put me in <title>'])`

Reply:
I would expect the `<title>` something similar to what `<h1>` contains, otherwise it is hard to distinguish different tabs.